doc/Doxyfile.in

```diff
@@ -10,9 +10,9 @@ INPUT                  = @PROJECT_SOURCE_DIR@/doc/index.doc \
                          @CMAKE_BINARY_DIR@/src/base/QXmppBuildConstants.h \
                          @PROJECT_SOURCE_DIR@/src
 HTML_EXTRA_FILES       = @PROJECT_SOURCE_DIR@/doc/doap.xml \
-                         @PROJECT_SOURCE_DIR@/doc/doap.xsl \
-                         @PROJECT_SOURCE_DIR@/doc/xeplist.xml \
-                         @PROJECT_SOURCE_DIR@/doc/doap.css
+                         @PROJECT_SOURCE_DIR@/doc/doap-style/doap.xsl \
+                         @PROJECT_SOURCE_DIR@/doc/doap-style/xeplist.xml \
```

Comment:
@lnjX `xeplist.xml` is no style. It only contains XEP metadata. Thus, the directory name should be adjusted accordingly.

Reply:
what's your suggestion?

Reply:
Maybe `doap`?

Reply:
Your intention was to separate the content, which is changed frequently by developers, from the other parts, right?

Reply:
yes, especially to separate the stuff for generating the styled xep list

Reply:
the `doap` folder doesn't contain the doap.xml, that could be misleading

Reply:
I see. How about `doap-externals`?

Reply:
Or `doap-rendering`?